src/workloads/query/GroupSpillToDisk.yml

```diff
@@ -129,14 +129,16 @@ AutoRun:
 - When:
     mongodb_setup:
       $eq:
-      # TODO PERF-3480: Enable this workload against 'standalone', 'standalone-all-feature-flags'
-      # and 'standalone-sbe' once the SBE performance improves. These queries will use SBE unless
-      # classic is forced, but they run for too long at the moment due to the performance problem
-      # reported in SERVER-70395.
+      - standalone
+      - standalone-all-feature-flags
       - standalone-classic-query-engine
+      - standalone-sbe
     branch_name:
       $neq:
       - v4.0
       - v4.2
       - v4.4
       - v5.0
+      - v6.0
```

Comment:
This benchmark cannot run against SBE on 6.0, 6.1, or 6.2 because I want to merge this patch before [SERVER-70395](https://jira.mongodb.org/browse/SERVER-70395) is backported. If we so choose, we can enable the benchmarks on 6.0 as part of a future change.

Reply:
Ack. Could you leave a note on https://jira.mongodb.org/browse/BACKPORT-14096 about this?

Reply:
Good idea. Done!